react/src/main.js

```diff
@@ -0,0 +1,11 @@
+import 'babel-polyfill';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './app';
+
+$(function() {
+  ReactDOM.render(
+    <App />,
+    document.getElementById('app')
```

Comment:
This was copy/pasted from the React configuration article on Horizon. I'm not sure if Hound is correct or the Horizon article is correct?